alembic/versions/bba80d21c5a4_.py

```diff
@@ -0,0 +1,63 @@
+"""Add tokens to database
+
+Revision ID: bba80d21c5a4
+Revises: b26ab0051a88
+Create Date: 2022-01-19 22:19:59.514998
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "bba80d21c5a4"
+down_revision = "b26ab0051a88"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute(
+        "INSERT INTO tokens (token_address,decimals) VALUES ('0x514910771af9ca656af840dff83e8264ecf986ca',18)"
```

Comment:
these can be one big statement

op.execute(
    """
    INSERT INTO tokens (token_address,decimals) VALUES 
    ('0x514910771af9ca656af840dff83e8264ecf986ca',18),
    ('0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',18)
    """
)